looty/src/main/scala/looty/model/ComputedItem.scala

```diff
@@ -42,7 +42,7 @@ class ComputedItem(val item: AnyItem, val containerId: LootContainerId, val loca
 
   object Scores {
     lazy val default: ItemScore = ItemScorer(ComputedItem.this).getOrElse(ItemScore(Nil, 0))
-    lazy val custom : ItemScore = ItemScorer(ComputedItem.this).getOrElse(ItemScore(Nil, 0))
+    lazy val custom : HighScore = HighScorer(ComputedItem.this).getOrElse(HighScore(Nil, 0))
```

Comment:
could you make this as a new custom2 field and then we can reference that next to where custom gets it's column from? ItemScorer is pretty outdated and there is still default to fallback on so no worries if not.